examples/inbound-deliveries-to-sap/INBOUND DELIVERIES TO SAP.md

```diff
@@ -0,0 +1,50 @@
+# Extract delivery information from XML files and confirm receipt in S/4HANA inbound delivery
+
+This example illustrates leveraging the `sap.s4hana.api_inbound_delivery_srv_0002:Client` in Ballerina for S/4HANA API interactions. It specifically showcases how to extract everyday delivery information from XML files and confirm the receipt in the S/4HANA inbound delivery.
+
+## Prerequisites
+
+### 1. Setup the S/4HANA API
+
+Refer to the [Setup Guide](https://central.ballerina.io/ballerinax/sap/latest#setup-guide) for necessary credentials (
+hostname, username, password).
+
+### 2. Include XML files inside the package path that follow the format below
```

Comment:
Lets give this under resources folder

Reply:
Updated in [Update reviewed changes](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.delivery/pull/16/commits/8650b99db9b8e7a6078a7120025cc0bf81979c17)